openage/convert/tool/interactive.py

```diff
@@ -29,13 +27,6 @@ def interactive_browser(cfg, srcdir=None):
     # the variables are actually used, in the interactive prompt.
     # pylint: disable=possibly-unused-variable
 
-    # Initialize game versions data
-
-    auxiliary_files_dir = cfg / "converter" / "games"
-    avail_game_eds, avail_game_exps = create_version_objects(auxiliary_files_dir)
-
-    # Acquire game version info
-    game_version = get_game_version(srcdir, avail_game_eds, avail_game_exps)
```

Comment:
it's good that this is now detected one layer above, we should try the same for other usages of the game version information (regarding the comment above i made).

Reply:
vcwf